testing/dart/spirv_exception_test.dart

```diff
@@ -0,0 +1,55 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.6
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:spirv/spirv.dart' as spirv;
+import 'package:path/path.dart' as path;
+import 'package:test/test.dart';
```

Comment:
This will need updates to account for https://github.com/flutter/engine/pull/26215